batch/gcp-create-worker-image.sh

```diff
@@ -24,7 +24,7 @@ else
     BUILDER=build-batch-worker-$NAMESPACE-image
```

Comment:
@sjparsa , I don't see any dev docs about how to update a worker image. Did someone explain how to do that to you?

I confirmed that batch-worker-13 already exists:
```
(base) dking@wm28c-761 hail % gcloud compute images list --project hail-vdc
NAME                                                  PROJECT              FAMILY                             DEPRECATED  STATUS
...
batch-worker-13                                       hail-vdc                                                            READY
```

Reply:
Yes Daniel talked to me about it! I asked him to make batch-worker-13 with the update so I think that should be why it's showing up? @daniel-goldstein

Reply:
@daniel-goldstein can you also create PR adding a dev-doc for how to do this? TY!